modules/impact/util_nodes.py

```diff
@@ -196,7 +196,7 @@ class ImpactLogger:
     @classmethod
     def INPUT_TYPES(s):
         return {"required": {
-                        "data": (any_typ, ""),
```

Comment:
I am not sure if it means to be a default value. If so, then change to {default: ""}